src/lib/notifications/backgroundFetch.native.js

```diff
@@ -30,91 +30,101 @@ const options = {
 
 const log = logger.child({ from: 'backgroundFetch' })
 
-export const initBGFetch = once((goodWallet, userStorage) => {
-  const task = async taskId => {
-    if (taskId === MAIN_TASK) {
-      log.info('[BackgroundFetch] taskId: ', taskId)
+const dailyClaimNotification = async goodWallet => {
+  const { entitlement: dailyUBI } = await goodWallet.getClaimScreenStatsFuse()
 
-      const isLoggedIn = await AsyncStorage.getItem(IS_LOGGED_IN)
-      const oldNotifications = await AsyncStorage.getItem(OLD_NOTIFICATIONS)
+  // We should notify once: only in first bg-fetch call after daily claim time.
+  // notificationTimeout is time of next bg-fetch call:
+  const notificationTimeout = dailyClaimTime + bgFetchFrequency
+  const needToNotify = dailyUBI && Date.now() >= dailyClaimTime && Date.now() < Date.now(notificationTimeout)
```

Comment:
replace this with a properties/localstorage variable
keep the last claim notification time. 
if dailyUBI && Date.now() >= dailyClaimTime && lastClaimNotification< dailyclaimtime
then you can notify